AzureDevOps.WikiPDFExport/WikiPDFExporter.cs

```diff
@@ -638,7 +631,7 @@ internal List<string> CreateGlobalTableOfContent(List<string> contents)
 
             foreach (var content in filteredContentList) 
             {
-                var headerMatches = Regex.Matches(content, "^ *#+ ?.*$", RegexOptions.Multiline);
+                var headerMatches = Regex.Matches(content, "^ *#{1,2} ?[^#].*$", RegexOptions.Multiline);
```

Comment:
```suggestion
                var headerMatches = Regex.Matches(content, "^ *#+ ?[^#].*$", RegexOptions.Multiline);
```
Revert the limit on the number of headings... was limiting my own doc. Should make that configurable in future.